### DISCUSSION TITLE
[Question][Hw1]

### DISCUSSION THREAD

**Discussion by anisrashidov** [prosyslab-classroom/cs348-information-security#664] [Q&A]
Name: Anis Rashidov

For Caesar Cipher, do we have to encode space and punctuation marks? If yes, then how should we do so?

**Comment by KalasLavas** [comment]
I removed all non-alphabetic characters, converted to correct case (uppercase for cipher and lowercase for plain) and it passed the tests.

**Comment by sujin0529** [comment]
Hi,

All the other characters (e.g., numbers and whitespaces) will be ignored in text preprocessing.
So, you don't need to consider other character encoding.

Thanks.

**Comment by anisrashidov** [comment]
Thanks all.
